.goreleaser.yml

```diff
@@ -0,0 +1,37 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  -
+    binary: biscuit
+    goos:
+      - darwin
+      - linux
+      - windows
+    goarch:
+      - amd64
+      - arm64
+      - 386
+    ldflags:
+      - -s -w
+      - -X main.Version={{.Version}}
+universal_binaries:
+  - replace: false
```

Comment:
Not replacing to that people can choose to download a smaller binary if they want, or use the fat binary for all architectures.